src/components/common/TravelTipsCard/TravelTipsCard.jsx

```diff
@@ -0,0 +1,60 @@
+import { memo, useState } from 'react';
+import { Link } from 'react-router-dom';
+import TravelTipsStub from '../../../assets/img/TravelTips/TravelTipsStub.png';
+import Heart from './../../../assets/img/TravelTips/heart.svg';
+import ReadMore from './../../../assets/img/TravelTips/read_more.svg';
+
+const TravelTipsCard = ({ tipHeader, tipText, date, image }) => {
+  const [likesAmount, setLikesAmount] = useState(0);
+  const [isLiked, setIsLiked] = useState(false);
+
+  const getShortText = (text, maxLength = 320) =>
```

Comment:
First comment, why do you need this function inside the component body if it accepts all required inputs as params?

Reply:
Second comment, I don't like the idea of text truncation in js. Try to make it using CSS, here is the article that might help https://www.freecodecamp.org/news/how-to-truncate-text-with-css-javascript/

Truncating text this way isn't reliable because it doesn't take into account styles of text (e.g. too big font size or something else)

Reply:
Yeah, you're damn right! Found the Line Clamp class. 
It truncates a block of text after a specific number of lines.
**God bless Tailwind!**